lib/config/config.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"text/tabwriter"
+)
+
+type Config struct {
+	Debug              bool            `json:"debug"`
+	LowCreditThreshold float64         `json:"lowCreditThreshold"`
+	Backend            string          `json:"backend"`
+	ListenPort         int             `json:"listenPort"`
+	LogFile            string          `json:"logFile"`
+	StatsFile          string          `json:"statsFile"`
+	Graphite           string          `json:"graphite"`
+	GraphitePrefix     string          `json:"graphitePrefix"`
+	TrustedProxies     string          `json:"trustedProxies"`
+	Slash32Share       float64         `json:"slash32Share"`
+	Slash24Share       float64         `json:"slash24Share"`
+	Slash16Share       float64         `json:"slash16Share"`
+	UserAgentShare     float64         `json:"userAgentShare"`
+	HashMaxLen         int             `json:"hashMaxLen"`
+	GraphiteURL        *url.URL        `json:"-"`
```

Comment:
`json:"-"` signals to the json marshaller and unmarshaller that it should ignore this property